README.md

```diff
@@ -1,99 +1,139 @@
-```
+* [Spark RAPIDS](https://github.com/NVIDIA/spark-rapids): A GPU accelerator plugin for [Apache Spark](https://spark.apache.org/)
+* [Velox-cuDF](https://github.com/facebookincubator/velox/blob/main/velox/experimental/cudf/README.md): A [Velox](https://velox-lib.io/)
+extension module to execute Velox plans on the GPU
+* [Sirius](https://www.sirius-db.com/): A GPU-native SQL engine providing extensions for libraries like [DuckDB](https://duckdb.org/)
 
-Or, you can use cuDF as a no-code-change accelerator for pandas, using
-[`cudf.pandas`](https://docs.rapids.ai/api/cudf/stable/cudf_pandas).
-`cudf.pandas` supports 100% of the pandas API, utilizing cuDF for
-supported operations and falling back to pandas when needed:
+## Installation
 
-```python
-%load_ext cudf.pandas  # pandas operations now use the GPU!
+### System Requirements
 
-import pandas as pd
+Operating System, GPU driver, and supported CUDA version information can be found at the [RAPIDS Installation Guide](https://docs.rapids.ai/install/#system-req)
 
-tips_df = pd.read_csv("https://github.com/plotly/datasets/raw/master/tips.csv")
-tips_df["tip_percentage"] = tips_df["tip"] / tips_df["total_bill"] * 100
+### pip
+
+Each cudf library is available on PyPI. You will need to match the major version number of your installed CUDA version with a `-cu##` suffix
+when installing from PyPI.
+
+```bash
+# CUDA 13
+pip install libcudf-cu13
+pip install pylibcudf-cu13
+pip install cudf-cu13
+pip install cudf-polars-cu13
+pip install dask-cudf-cu13
 
-# display average tip by dining party size
-print(tips_df.groupby("size").tip_percentage.mean())
+# CUDA 12
+pip install libcudf-cu12
+pip install pylibcudf-cu12
+pip install cudf-cu12
+pip install cudf-polars-cu12
+pip install dask-cudf-cu12
 ```
 
-## Resources
+### conda
 
-- [Try cudf.pandas now](https://nvda.ws/rapids-cudf): Explore `cudf.pandas` on a free GPU enabled instance on Google Colab!
-- [Install](https://docs.rapids.ai/install): Instructions for installing cuDF and other [RAPIDS](https://rapids.ai) libraries.
-- [cudf (Python) documentation](https://docs.rapids.ai/api/cudf/stable/)
-- [libcudf (C++/CUDA) documentation](https://docs.rapids.ai/api/libcudf/stable/)
-- [RAPIDS Community](https://rapids.ai/learn-more/#get-involved): Get help, contribute, and collaborate.
+Each cudf library is available to be installed with the conda package manager by specifying the `-c rapidsai` channel.
 
-See the [RAPIDS install page](https://docs.rapids.ai/install) for
-the most up-to-date information and commands for installing cuDF
-and other RAPIDS packages.
+```bash
+conda install -c rapidsai libcudf
+conda install -c rapidsai pylibcudf
+conda install -c rapidsai cudf
+conda install -c rapidsai cudf-polars
+conda install -c rapidsai dask-cudf
+```
 
-## Installation
+A development version of each library is available as a nightly release by specifying the `-c rapidsai-nightly` channel instead.
```

Comment:
If we're mentioning development versions, we should also mention the nightly wheels https://anaconda.org/rapidsai-wheels-nightly/repo?type=pypi&label=main